templates/fine-tune-llm_v2/cookbooks/customize_initial_weights_and_prompt_format/llama-guard-2.yaml

```diff
@@ -0,0 +1,48 @@
+model_id: meta-llama/Meta-Llama-3-8B # <-- change this to the model with the same architecture as the model weights you will bring
```

Comment:
Let's make this yaml ready to run. Like with real uploaded weights and real formatted data. no placeholders.

Reply:
Done. I've uploaded llama guard 2 weights to our regular S3 mirror